include/ebpf_result.hpp

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) Microsoft Corporation
```

Comment:
Why the hpp extension? This is usually reserved for headers with C++ specific features (i.e. that can't be use from a .c file).

Reply:
Renamed.